packages/form-builder/src/types/formSettings.ts

```diff
@@ -22,4 +23,10 @@ export type FormSettings = {
     receiptHeading: string;
     receiptDescription: string;
     formStatus: FormStatus;
+    emailTemplateOptions: EmailTemplateOption[];
+    emailOptionsStatus: string;
+    emailTemplate: string;
+    emailLogo: string;
+    emailFromName: string;
+    emailFromEmail: string;
```

Comment:
As we get into these settings with a lot of options, we might consider nesting them into their own object to simplify things like:

```
formSettings: {
  email: {
    templateOptions,
    optionStatus,
    template,
    logo,
    fromName,
    fromEmail
  }
}
```

Reply:
I started working through this, but in practice it felt more complicated and hard to read when using the settings.